pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -69,6 +69,12 @@ type APIExport struct {
 	Status APIExportStatus `json:"status,omitempty"`
 }
 
+const (
+	// AnnotationAPIExportExtraKeyPrefix is the prefix of an annotation set on an APIExport to provide extra info. The
+	// annotation with the prefix will be synced to all the APIBindings bound to this APIExport.
+	AnnotationAPIExportExtraKeyPrefix = "extra.apis.kcp.dev/"
```

Comment:
copied once or continously synced? If the later, can I remove?

Reply:
continuously synced

Reply:
And yes, you can remove it.

Reply:
Please clarify in the description.